Patch by Robert Scheck <robert@fedoraproject.org> for dsniff >= 2.4b1 which
allows building against libtirpc (as a separate library) given the Sun RPC
support in glibc has been deprecated for a long time.

--- dsniff-2.4/Makefile.in		2018-05-26 22:04:34.809966900 +0200
+++ dsniff-2.4/Makefile.in.libtirpc	2018-05-27 02:56:31.149313503 +0200
@@ -16,7 +16,7 @@
 mandir		= @mandir@
 
 CC	= @CC@
-CFLAGS	= @CFLAGS@ -DDSNIFF_LIBDIR=\"$(libdir)/\"
+CFLAGS	= @CFLAGS@ -DDSNIFF_LIBDIR=\"$(libdir)/\" -I/usr/include/tirpc
 LDFLAGS	= @LDFLAGS@
 
 PCAPINC = @PCAPINC@
@@ -93,7 +93,7 @@
 	$(RANLIB) $@
 
 dsniff: $(HDRS) $(SRCS) $(OBJS) libmissing.a
-	$(CC) $(LDFLAGS) -o $@ $(OBJS) $(LIBS) $(NIDSLIB) $(PCAPLIB) $(LNETLIB) $(DBLIB) $(SSLLIB)
+	$(CC) $(LDFLAGS) -o $@ $(OBJS) $(LIBS) $(NIDSLIB) $(PCAPLIB) $(LNETLIB) $(DBLIB) $(SSLLIB) -ltirpc
 
 arpspoof: arpspoof.o arp.o libmissing.a
 	$(CC) $(LDFLAGS) -o $@ arpspoof.o arp.o $(LIBS) $(PCAPLIB) $(LNETLIB)
@@ -102,7 +102,7 @@
 	$(CC) $(LDFLAGS) -o $@ dnsspoof.o pcaputil.o $(LIBS) $(PCAPLIB) $(LNETLIB)
 
 filesnarf: nfs_prot.o filesnarf.o pcaputil.o rpc.o libmissing.a
-	$(CC) $(LDFLAGS) -o $@ filesnarf.o nfs_prot.o pcaputil.o rpc.o $(LIBS) $(NIDSLIB) $(PCAPLIB) $(LNETLIB)
+	$(CC) $(LDFLAGS) -o $@ filesnarf.o nfs_prot.o pcaputil.o rpc.o $(LIBS) $(NIDSLIB) $(PCAPLIB) $(LNETLIB) -ltirpc
 
 macof: macof.o libmissing.a
 	$(CC) $(LDFLAGS) -o $@ macof.o $(LIBS) $(PCAPLIB) $(LNETLIB)
@@ -114,7 +114,7 @@
 	$(CC) $(LDFLAGS) -o $@ msgsnarf.o buf.o pcaputil.o $(LIBS) $(NIDSLIB) $(PCAPLIB) $(LNETLIB)
 
 sshmitm: sshmitm.o buf.o hex.o record.o ssh.o sshcrypto.o libmissing.a
-	$(CC) $(LDFLAGS) -o $@ sshmitm.o buf.o hex.o record.o ssh.o sshcrypto.o $(LIBS) $(LNETLIB) $(DBLIB) $(SSLLIB)
+	$(CC) $(LDFLAGS) -o $@ sshmitm.o buf.o hex.o record.o ssh.o sshcrypto.o $(LIBS) $(LNETLIB) $(DBLIB) $(SSLLIB) -ltirpc
 
 sshow: sshow.o pcaputil.o libmissing.a
 	$(CC) $(LDFLAGS) -o $@ sshow.o pcaputil.o $(LIBS) $(NIDSLIB) $(PCAPLIB) $(LNETLIB)
@@ -132,7 +132,7 @@
 	$(CC) $(LDFLAGS) -o $@ urlsnarf.o base64.o buf.o pcaputil.o $(LIBS) $(NIDSLIB) $(PCAPLIB) $(LNETLIB)
 
 webmitm: webmitm.o base64.o buf.o decode_http.o record.o libmissing.a
-	$(CC) $(LDFLAGS) -o $@ webmitm.o base64.o buf.o decode_http.o record.o $(LIBS) $(LNETLIB) $(DBLIB) $(SSLLIB)
+	$(CC) $(LDFLAGS) -o $@ webmitm.o base64.o buf.o decode_http.o record.o $(LIBS) $(LNETLIB) $(DBLIB) $(SSLLIB) -ltirpc
 
 webspy: webspy.o base64.o buf.o remote.o libmissing.a
 	$(CC) $(LDFLAGS) -o $@ webspy.o base64.o buf.o remote.o $(LIBS) $(NIDSLIB) $(PCAPLIB) $(LNETLIB) $(X11LIB)
